src/main/java/duke/Main.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.controller.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke();
+
+    /**
+     * Sets up program to run MainWindow on start
+     *
+     * @param stage Stage to show
```

Comment:
perhaps this comment is incomplete? im not too sure what you mean by "Stage to show"